pkg/landscaper/installations/executions/template/gotemplate.go

```diff
@@ -480,3 +483,23 @@ func resolveComponents(defaultCD *cdv2.ComponentDescriptor, list *cdv2.Component
 
 	return components, nil
 }
+
+func getImageVectorGoFunc(cd *cdv2.ComponentDescriptor, list *cdv2.ComponentDescriptorList) func(args ...interface{}) string {
+	return func(args ...interface{}) string {
+		if cd == nil {
```

Comment:
can you make the component descriptor and the list configurable and default to `cd` and `list`.
so that we have a parameter for the component descriptor and list

Reply:
we made the arguments configurable